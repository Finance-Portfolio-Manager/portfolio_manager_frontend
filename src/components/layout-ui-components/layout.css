@import url('https://fonts.googleapis.com/css2?family=Fira+Sans&family=Inter:wght@700&display=swap');
:root{
    --darkmode-dark-bg: #151B2C;
    --darkmode-med-bg: #21293E;
    color: #000 !important;
    font-family: 'Inter', sans-serif;
    font-family: 'Fira Sans', sans-serif;

}

.layout-background{
    background-color: var(--darkmode-dark-bg) !important;
    min-height: 100vh !important;

}

.card-body{
    background-color: var(--darkmode-med-bg) !important;
    margin: 10px !important;
    border-radius: 25px !important;
    text-align: center;
    box-shadow: -6px 0px 6px rgba(0, 0, 0, 0.11),  6px 0px 6px rgba(0, 0, 0, 0.11) ;
}

.card-header{
    color: white;
    font-family: "Inter", sans-serif;
    
}

.chart-container{
    margin-bottom: 10px;
}

.styled-text{
    font-family: 'Fira Sans' !important;
    color: white !important;
}

.nav-temp{
    background-color: var(--darkmode-med-bg) !important;
    box-shadow: -6px 0px 6px rgba(0, 0, 0, 0.11),  6px 0px 6px rgba(0, 0, 0, 0.11) ;
}
.zoom{
    transition: transform .3s
}

  
.zoom:hover {
    
    transform: scale(1.05); /* (150% zoom - Note: if the zoom is too large, it will go outside of the viewport) */
  }

.grid-container{
    justify-content: space-evenly;
}