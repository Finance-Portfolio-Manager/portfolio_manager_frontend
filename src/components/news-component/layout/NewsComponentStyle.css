@import url('https://fonts.googleapis.com/css2?family=Archivo+Black&display=swap');
:root {
  --dark-cosmos-color: rgb(31, 17, 34);


  /* The poster Illustration colors */
  --poster: rgb(224, 198, 178);
  --poster-border: min(3vmax, 50px);


  --mq-small: 650px;
  --mq-medium: 850px;
  --mq-vert-small: 650px;
}

/* .news__card-body {
    position: relative;
    z-index: 100;
    border: var(--poster-border) solid;
    display: flex;
    flex-flow: column;
    justify-content: center;
} */
@media (min-width: var(--mq-small)) {
    .news__card-body {
        height: 50vh;
        width: 50%;
   }
}
@media (min-width: var(--mq-medium)) {
    .news__card-body {
        height: 65vh;
        width: 50%;
        max-width: 600px;
        margin-top: 0;
   }
}
@media (min-width: var(--mq-small)) and (max-height: var(--mq-small)) {
    .news__card-body {
        --poster-border: 30px;
        border: var(--poster-border) solid;
        border-top: 8px solid;
        height: 80vh;
        width: 80vw;
   }
}


.news__card-heading, .news__card-explanation {
    background: var(--poster);
    box-shadow: 0 0 0px 1px var(--poster);
}


.smallnumber {
    font-size: 0.8em;
    text-align: right;
    position: relative;
    top: calc(0px - var(--poster-border) + 25px);
    right: calc(0px - var(--poster-border) + 15px);
}
@media (min-width: var(--mq-small)) {
    .smallnumber {
        font-size: 1em;
        top: calc(0px - var(--poster-border) + 20px);
        right: calc(0px - var(--poster-border) + 10px);
   }
}
@media (min-width: var(--mq-medium)) {
    .smallnumber {
        font-size: 1.3em;
        top: calc(0px - var(--poster-border) + 30px);
        right: calc(0px - var(--poster-border) + 20px);
   }
}
@media (min-width: var(--mq-small)) and (max-height: var(--mq-small)) {
    .smallnumber {
        font-size: 1em;
        top: 7px;
        right: calc(0px - var(--poster-border) + 10px);
   }
}


/* h1, */
.smallnumber {
  font-family: "Archivo Black", sans-serif;
  margin: 0;
}

.news_header-one {
    font-size: clamp(1.8em, 3vw, 3.2em);
    letter-spacing: -0.03em;
    margin-bottom: 0;
    margin-top: -0.3em;
    position: relative;
    bottom: -0.22em;
}
@media (min-width: var(--mq-small)) {
    .news_header-one {
        bottom: -0.2em;
   }
}
@media (min-width: var(--mq-small)) and (max-height: var(--mq-small)) {
    .news_header-one {
        font-size: clamp(1.5em, 8vw, 2em);
   }
}


/* Explanation */
.news-card-explanation {
    padding: 3vh 0;
    font-size: 1em;
    line-height: 1.3;
    border-bottom: 3px solid var(--dark-cosmos-color);
    position: relative;
}
@media (min-width: var(--mq-medium)) {
    .news-card-explanation {
        font-size: 1.1em;
        line-height: 1.4;
        padding: 3vh 0;
        margin-bottom: -0.9em;
        border-bottom: 6px solid var(--dark-cosmos-color);
   }
    .news-card-explanation:before {
        content: "";
        position: absolute;
        top: -6vh;
        left: -1.2vh;
        width: 2vh;
        height: 5vh;
        background-repeat: no-repeat;
        background-size: 100%;
   }
    .news-card-explanation:before {
        top: -8vw;
        left: -1.3vw;
        width: 2.5vw;
        height: 16vw;
   }
}
@media (min-width: var(--mq-small)) and (max-height: var(--mq-small)) {
    .news-card-explanation {
        font-size: 1em;
        border-bottom: 2px solid var(--dark-cosmos-color);
   }
    .news-card-explanation:before {
        background: none;
   }
}


/* 
Hero Section CSS 
file created by carlos galvan jr
*/
.news__hero-section {
  color: #fff;
  padding: 160px 0;
  min-height: 100vh;
}

.news__hero-row {
  align-items: center;
}

.news_row {
  display: flex;
  margin-right: -15px;
  margin-bottom: -15px;
  margin-left: -15px;
  flex-wrap: wrap;
  align-content: stretch;
}

.news_col {
  margin-bottom: 15px;
  padding-right: 15px;
  padding-left: 15px;
  flex: 1;
  max-width: 50%;
  flex-basis: 50%;
}

.news__hero-text-wrapper {
  max-width: 540px;
  padding-top: 0;
  padding-bottom: 60px;
}

.top-line {
  /* color: #f00946; */
  font-size: 16px;
  line-height: 16px;
  font-weight: 700;
  letter-spacing: 1.4px;
  text-transform: uppercase;
  margin-bottom: 16px;
}

.heading {
  margin-bottom: 24px;
  font-size: 48px;
  line-height: 1.1;
  font-weight: 600;
  color: #f7f8fa;
}

.dark {
  color: #1c2237;
}

.darkBg {
  background-color: #1c2237;
}

.news__hero-subtitle {
  max-width: 440px;
  margin-bottom: 35px;
  font-size: 18px;
  line-height: 24px;
}

.news__hero-img-wrapper {
  max-width: 555px;
}

.news__hero-img {
  max-width: 95%;
  margin-top: 0;
  margin-right: 0;
  margin-left: 10px;
  padding-right: 0;
}

img {
  border: 0;
  max-width: 100%;
  vertical-align: middle;
  display: inline-block;
}

@media screen and (max-width: 991px) {
  .container {
    padding-right: 30px;
    padding-left: 30px;
  }
}

@media screen and (max-width: 768px) {
  .news__hero-text-wrapper {
    padding-bottom: 65px;
  }

  .col {
    max-width: 100%;
    flex-basis: 100%;
  }
}
