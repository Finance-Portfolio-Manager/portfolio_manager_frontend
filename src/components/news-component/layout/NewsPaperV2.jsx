import React from "react";
import "../../../css/news-flex.css";
import { useState } from "react";
import { GrNext, GrPrevious } from "react-icons/gr";
export const NewsPaperV2 = (props) => {
  //   const [story, setStory] = useState(props.story);
  return (
    <div className="news__body primary-text global__link">
      <div className="news__container ">
        <div className="news__title-container global__gradient-background">
          <div className="news__center">
            <div className="news__title-header news__zoom global__gradient-text">
              {props.story.title}
            </div>
            <div className="news__source-container ">
              <a href={props.story.url} className="news__source-link ">
                Source
              </a>
            </div>
          </div>
          <div className="news__next-button-v2 ">
            <button
              onClick={props.incrementIndex}
              className="news__button primary-text"
              aria-label='next news article'
            >
              <GrNext className="primary-text" />
            </button>
          </div>
          <div className="news__previous-button-v2">
            <button
              onClick={props.decrementIndex}
              aria-label='previous news article'
            >
              <GrPrevious />
            </button>
          </div>
        </div>
        <div className="news__story-container">
          <div className="news__row">
            <div className="news__col news__zoom">
              <div className="news__text-wrapper">
                <p className="news__description">{props.story.description}</p>
                <div className="news__description-card-footer">
                  <a href={props.story.url} className="news__readmore">
                    read more
                  </a>
                </div>
              </div>
            </div>
            <div className="news__col news__zoom">
              <div className="news__image-wrapper">
                <img
                  src={props.story.urlToImage}
                  alt="news article image"
                  className="news__img"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  );
};
