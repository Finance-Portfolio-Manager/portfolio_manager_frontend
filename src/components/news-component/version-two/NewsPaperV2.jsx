import React from "react";
import "./news-flex.css";
import { useState } from "react";
export const NewsPaperV2 = (props) => {
//   const [story, setStory] = useState(props.story);
  return (
    <div className="news__body primary-text">
      <div className="news__title-container global__gradient-background">
        <div className="news__center">
          <div className="news__title-header news__zoom global__gradient-text" >{props.story.title}</div>
          <div className="news__source-container">
            <a href={props.story.url} className="news__source-link">
              Source
            </a>
          </div>
        </div>
    
      </div>
      <div className="news__story-container news__story-center">
        <div className="news__description-card news__zoom">
          <p className="news__description">{props.story.description}</p>
          <div className="news__description-card-footer">
            <a href={props.story.url} className="news__readmore">
              read more
            </a>
          </div>
        </div>
        <div className="news__image-wrapper news__zoom">
          <img src={props.story.urlToImage} className="news__img" />
        </div>
      </div>
    </div>
  );
};
