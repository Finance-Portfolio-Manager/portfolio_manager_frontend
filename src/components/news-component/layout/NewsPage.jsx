/*
 Carlos Galvan Jr + Greg McCoy

 This is the container component that fetches the news and uses the NewsPaperV2 component to format
  TO DO:
  Add key binds
  implement data and author
  fix links
  fix mobiles
*/

import React, { useState, useEffect } from "react";
import axios from "axios";
import { NewsPaperV2 } from "./NewsPaperV2";
// import { GrNext, GrPrevious } from "react-icons/gr";

export const NewsPage = () => {
  const [news, setNews] = useState([]);
  const [index, setIndex] = useState(0);

  const getNews = () => {
    axios
      .get("http://23.22.140.95:8082/news/get-news")
      .then((response) => {
        setNews(response.data);
      })
      .catch((error) => console.log(`Error ${error}`));
  };
 
  useEffect(() => {
    getNews();
  }, [setIndex]);

  const incrementIndex = () =>{
    setIndex((index + 1) % news.length);
  }
  const decrementIndex = () =>{
    setIndex((index - 1 + news?.length) % news?.length);
  }

  return (
    <>
      {news?.length ? (
        <>
          {/* <div className="news__next-button-v2 news__button">
            <button
              onClick={incrementIndex}
              className="news__button primary-text"
              aria-label='next news article'
            >
              <GrNext className="primary-text" />
            </button>
          </div>
          <div className="news__previous-button-v2 news__button">
            <button
              onClick={decrementIndex}
              aria-label='previous news article'
            >
              <GrPrevious />
            </button>
          </div> */}

          <NewsPaperV2 key={news[index]?.url} story={news[index]} incrementIndex={incrementIndex} decrementIndex={decrementIndex} />
        </>
      ) : (
        <div className="news__empty">
          <div className="news__empty-header">
            No news yet... Try again later
          </div>
        </div>
      )}
    </>
  );
};
