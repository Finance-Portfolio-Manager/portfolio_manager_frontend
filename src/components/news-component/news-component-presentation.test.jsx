import { NewsPage } from "./layout/NewsPage";
import React from "react";
import { NewsPaperV2 } from "./layout/NewsPaperV2";
import { mount, shallow } from "enzyme";
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { act } from 'react-dom/test-utils';
import ReactDOM from 'react-dom';
import renderer from "react-test-renderer";
import { render } from "@testing-library/react";
import { GrNext } from "react-icons/gr";
import userEvent from "@testing-library/user-event";
// import { useEffect } from "react";

configure({adapter: new Adapter() });

describe("Testing News Component", ()=>{

    // let wrapperWithNewss;

    // beforeEach(() => {
    //     useEffect = jest.spyOn(React, "useEffect").mockImplementation(f => f());

    //     const mockData = [
    //         {"title": "Facebook Ranked Among Today’s Trending Stocks", "description": "Q.ai runs daily factor models to get the most up-to-date reading on stocks and ETFs. Today, our deep-learning algorithms have identified Facebook among others.", "url": "https://www.forbes.com/sites/qai/2021/08/09/facebook-ranked-among-todays-trending-stocks/", "urlToImage": "https://thumbor.forbes.com/thumbor/fit-in/1200x0/https%3A%2F%2Fspecials-images.forbesimg.com%2Fimageserve%2F61112c3daa5906c61c419c4e%2F0x0.png"},
    //         {"title": "Netflix Ranked Among Today’s Trending Stocks", "description": "Q.ai runs daily factor models to get the most up-to-date reading on stocks and ETFs. Today, our deep-learning algorithms have identified Facebook among others.", "url": "https://www.forbes.com/sites/qai/2021/08/09/facebook-ranked-among-todays-trending-stocks/", "urlToImage": "https://thumbor.forbes.com/thumbor/fit-in/1200x0/https%3A%2F%2Fspecials-images.forbesimg.com%2Fimageserve%2F61112c3daa5906c61c419c4e%2F0x0.png"},
    //         {"title": "Instagram Ranked Among Today’s Trending Stocks", "description": "Q.ai runs daily factor models to get the most up-to-date reading on stocks and ETFs. Today, our deep-learning algorithms have identified Facebook among others.", "url": "https://www.forbes.com/sites/qai/2021/08/09/facebook-ranked-among-todays-trending-stocks/", "urlToImage": "https://thumbor.forbes.com/thumbor/fit-in/1200x0/https%3A%2F%2Fspecials-images.forbesimg.com%2Fimageserve%2F61112c3daa5906c61c419c4e%2F0x0.png"},
    //         {"title": "Amazon Ranked Among Today’s Trending Stocks", "description": "Q.ai runs daily factor models to get the most up-to-date reading on stocks and ETFs. Today, our deep-learning algorithms have identified Facebook among others.", "url": "https://www.forbes.com/sites/qai/2021/08/09/facebook-ranked-among-todays-trending-stocks/", "urlToImage": "https://thumbor.forbes.com/thumbor/fit-in/1200x0/https%3A%2F%2Fspecials-images.forbesimg.com%2Fimageserve%2F61112c3daa5906c61c419c4e%2F0x0.png"},
    //         {"title": "Snapchat Ranked Among Today’s Trending Stocks", "description": "Q.ai runs daily factor models to get the most up-to-date reading on stocks and ETFs. Today, our deep-learning algorithms have identified Facebook among others.", "url": "https://www.forbes.com/sites/qai/2021/08/09/facebook-ranked-among-todays-trending-stocks/", "urlToImage": "https://thumbor.forbes.com/thumbor/fit-in/1200x0/https%3A%2F%2Fspecials-images.forbesimg.com%2Fimageserve%2F61112c3daa5906c61c419c4e%2F0x0.png"}
    //     ];

    //     const wrapperWithNewss = mount(<NewsPage />);
    // });

    // afterEach(() => {
    // document.body.removeChild(container);
    // container = null;
    // });

    test("News component renders", () =>{
        const wrapper = shallow(<NewsPage />);
        expect(wrapper.exists()).toBe(true);
    });

    test("News component uses given info", () =>{

        const index = 0;

        const incrementIndex = () =>{
            setIndex((index + 1) % mockData.length);
          }
        const decrementIndex = () =>{
        setIndex((index - 1 + mockData?.length) % mockData?.length);
        }

        const mockData = [
            {"title": "Facebook Ranked Among Today’s Trending Stocks", "description": "Q.ai runs daily factor models to get the most up-to-date reading on stocks and ETFs. Today, our deep-learning algorithms have identified Facebook among others.", "url": "https://www.forbes.com/sites/qai/2021/08/09/facebook-ranked-among-todays-trending-stocks/", "urlToImage": "https://thumbor.forbes.com/thumbor/fit-in/1200x0/https%3A%2F%2Fspecials-images.forbesimg.com%2Fimageserve%2F61112c3daa5906c61c419c4e%2F0x0.png"},
            {"title": "Netflix Ranked Among Today’s Trending Stocks", "description": "Q.ai runs daily factor models to get the most up-to-date reading on stocks and ETFs. Today, our deep-learning algorithms have identified Facebook among others.", "url": "https://www.forbes.com/sites/qai/2021/08/09/facebook-ranked-among-todays-trending-stocks/", "urlToImage": "https://thumbor.forbes.com/thumbor/fit-in/1200x0/https%3A%2F%2Fspecials-images.forbesimg.com%2Fimageserve%2F61112c3daa5906c61c419c4e%2F0x0.png"},
            {"title": "Instagram Ranked Among Today’s Trending Stocks", "description": "Q.ai runs daily factor models to get the most up-to-date reading on stocks and ETFs. Today, our deep-learning algorithms have identified Facebook among others.", "url": "https://www.forbes.com/sites/qai/2021/08/09/facebook-ranked-among-todays-trending-stocks/", "urlToImage": "https://thumbor.forbes.com/thumbor/fit-in/1200x0/https%3A%2F%2Fspecials-images.forbesimg.com%2Fimageserve%2F61112c3daa5906c61c419c4e%2F0x0.png"},
            {"title": "Amazon Ranked Among Today’s Trending Stocks", "description": "Q.ai runs daily factor models to get the most up-to-date reading on stocks and ETFs. Today, our deep-learning algorithms have identified Facebook among others.", "url": "https://www.forbes.com/sites/qai/2021/08/09/facebook-ranked-among-todays-trending-stocks/", "urlToImage": "https://thumbor.forbes.com/thumbor/fit-in/1200x0/https%3A%2F%2Fspecials-images.forbesimg.com%2Fimageserve%2F61112c3daa5906c61c419c4e%2F0x0.png"},
            {"title": "Snapchat Ranked Among Today’s Trending Stocks", "description": "Q.ai runs daily factor models to get the most up-to-date reading on stocks and ETFs. Today, our deep-learning algorithms have identified Facebook among others.", "url": "https://www.forbes.com/sites/qai/2021/08/09/facebook-ranked-among-todays-trending-stocks/", "urlToImage": "https://thumbor.forbes.com/thumbor/fit-in/1200x0/https%3A%2F%2Fspecials-images.forbesimg.com%2Fimageserve%2F61112c3daa5906c61c419c4e%2F0x0.png"}
        ];
        const wrapperWithNews = shallow(<NewsPaperV2 key={mockData[index]?.url} story={mockData[index]} incrementIndex={incrementIndex} decrementIndex={decrementIndex}/>);
        expect(wrapperWithNews.exists()).toBe(true);
    })

    // test("Matching snapshot..", () => {
    //     // console.log(container.inn);
    //     // act(() => {
    //     //     ReactDOM.render(<NewsPage />, container);
    //     // });
    //     // // const tree = container.toJSON();
    //     // // console.log(container);
    //     expect(wrapperWithNewss).toMatchSnapshot();
    // })

    test("Not rendering News...", () =>{
        const wrapperWithNews = mount(<NewsPage />);
        const div = wrapperWithNews.find("div.news__empty-header");
        expect(div.text()).toContain("No news yet... Try again later");
    })

    // it("Testing slides", () => {
    //     // const changeIndex = jest.fn();
    //     // const wrapperWithNews = shallow(<NewsPage />);
    //     // const node = wrapperWithNews.getNode('NewsPaperV2');
    //     // console.log(node.type);
    //     // const handleClick = jest.spyOn(React, "useState");
    //     // handleClick.mockImplementation(index => [index, changeIndex]);

    //     // wrapperWithNews.find('.news__button').simulate("click");
    //     // expect(changeIndex).toBeTruthy();
    //     act(() => {
    //         ReactDOM.render(<NewsPage />, container);
    //       });
    //     const btn = container.querySelector("button.news_button");

    //     const index = 0;

    //     const incrementIndex = () =>{
    //         setIndex((index + 1) % mockData.length);
    //       }
    //     const decrementIndex = () =>{
    //         setIndex((index - 1 + mockData?.length) % mockData?.length);
    //     }

    //     const mockData = [
    //         {"title": "Facebook Ranked Among Today’s Trending Stocks", "description": "Q.ai runs daily factor models to get the most up-to-date reading on stocks and ETFs. Today, our deep-learning algorithms have identified Facebook among others.", "url": "https://www.forbes.com/sites/qai/2021/08/09/facebook-ranked-among-todays-trending-stocks/", "urlToImage": "https://thumbor.forbes.com/thumbor/fit-in/1200x0/https%3A%2F%2Fspecials-images.forbesimg.com%2Fimageserve%2F61112c3daa5906c61c419c4e%2F0x0.png"},
    //         {"title": "Netflix Ranked Among Today’s Trending Stocks", "description": "Q.ai runs daily factor models to get the most up-to-date reading on stocks and ETFs. Today, our deep-learning algorithms have identified Facebook among others.", "url": "https://www.forbes.com/sites/qai/2021/08/09/facebook-ranked-among-todays-trending-stocks/", "urlToImage": "https://thumbor.forbes.com/thumbor/fit-in/1200x0/https%3A%2F%2Fspecials-images.forbesimg.com%2Fimageserve%2F61112c3daa5906c61c419c4e%2F0x0.png"},
    //         {"title": "Instagram Ranked Among Today’s Trending Stocks", "description": "Q.ai runs daily factor models to get the most up-to-date reading on stocks and ETFs. Today, our deep-learning algorithms have identified Facebook among others.", "url": "https://www.forbes.com/sites/qai/2021/08/09/facebook-ranked-among-todays-trending-stocks/", "urlToImage": "https://thumbor.forbes.com/thumbor/fit-in/1200x0/https%3A%2F%2Fspecials-images.forbesimg.com%2Fimageserve%2F61112c3daa5906c61c419c4e%2F0x0.png"},
    //         {"title": "Amazon Ranked Among Today’s Trending Stocks", "description": "Q.ai runs daily factor models to get the most up-to-date reading on stocks and ETFs. Today, our deep-learning algorithms have identified Facebook among others.", "url": "https://www.forbes.com/sites/qai/2021/08/09/facebook-ranked-among-todays-trending-stocks/", "urlToImage": "https://thumbor.forbes.com/thumbor/fit-in/1200x0/https%3A%2F%2Fspecials-images.forbesimg.com%2Fimageserve%2F61112c3daa5906c61c419c4e%2F0x0.png"},
    //         {"title": "Snapchat Ranked Among Today’s Trending Stocks", "description": "Q.ai runs daily factor models to get the most up-to-date reading on stocks and ETFs. Today, our deep-learning algorithms have identified Facebook among others.", "url": "https://www.forbes.com/sites/qai/2021/08/09/facebook-ranked-among-todays-trending-stocks/", "urlToImage": "https://thumbor.forbes.com/thumbor/fit-in/1200x0/https%3A%2F%2Fspecials-images.forbesimg.com%2Fimageserve%2F61112c3daa5906c61c419c4e%2F0x0.png"}
    //     ];
    //     // const wrapperWithNews = mount(<NewsPage />);
    //     // wrapperWithNews.invoke('handler')();
    //     // const button = wrapperWithNews.find("button.news__button");
    //     const clickF = () =>{
    //         btn.simulate("click");
    //     }

    //     expect(clickF).not.toThrow();
    // })
});