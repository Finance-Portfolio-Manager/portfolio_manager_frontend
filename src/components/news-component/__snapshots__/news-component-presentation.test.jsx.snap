// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing News Component Matching snapshot.. 1`] = `
<div
  className="news__empty"
>
  <div
    className="news__empty-header"
  >
    No news yet... Try again later
  </div>
</div>
`;
